src/components/v5/common/Table/partials/BurgerMenu.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import Icon from '~shared/Icon';
+import { BurgerMenuProps } from '../types';
+import Card from '~v5/shared/Card';
+import { useDetectClickOutside } from '~hooks';
+
+const displayName = 'v5.common.Table.partials.BurgerMenu';
+
+const BurgerMenu: FC<BurgerMenuProps> = ({
+  onToogle,
+  onToogleOff,
+  isMenuVisible,
+  canRemoveRow = true,
+  canDuplicateRow = true,
+  onRemoveRow,
+  onDuplicateRow,
```

Comment:
Can you check if `onRemoveRow` or `onDuplicateRow` functions are passed instead of adding additional booleans?